### ISSUE TITLE
Idea to add a Read and Confirm prompt to the wallet to maybe avoid issues

### ISSUE THREAD

**Issue by Crypto-Keeper-miners** [litecoin-foundation/litewallet-android#102]
Add a READ and CONFIRM

To resolve some of the day-to-day issues users have.

Add a read and confirm section that basically says the following after they download the wallet.

1) Write down your paper key and store in a safe place as your coins are not on the wallet or your phone but on the Blockchain and in the event you lose your device you can recover your coins.
2) always make sure your wallet is fully synced to the blockchain in order to see and or do a transaction. We recommend logging in once a week to avoid long syncing times and issues.
3) IF YOU DONATE to the Litecoin Foundation, there are no refunds.
4) Write down your PIN as this is the method used to send and view your Paper Keys and keep in a safe place with your paper keys.
5) Please look over Settings and Security Center to complete your PIN LIMIT, Fingerprint or Facial Recognition settings etc.
Then a confirm button that then takes them to Write down your Paper Key and add PIN.

Also, with regards to the PIN. Having the ability to change it or see it if you lose or forget it. Maybe with a fingerprint or facial recognition confirmation.
This would eliminate the need to have their email or cell phone information or a 2FA app.

**Comment by Patrick-t-mcdermott** [comment]
Yes, I think allowing users to upgrade their pincode with Face-iD or Fingerprint is a great idea. It would prevent them from having to go through the long task of recovering their wallet from the seed words. That task could be daunting for many new and non-technical users. Updating a forgotten pincode should be easy as possible.

**Comment by kcw-grunt** [comment]
Guys we should look at some other wallets and see how they do it.  No need to re-invent the wheel.  There are so many.

**Comment by Crypto-Keeper-miners** [comment]
After looking at several of my wallets, Pin is secondary to fingerprint. Fingerprint overrides Pin. Even my bank app.

**Comment by kcw-grunt** [comment]
@Crypto-Keeper-miners there are about 4 features in your description.  What we like to do is break each feature into a separate issue.  That way if there is a bug it's isolated.
So, the donation confirm is already merged: #60 
Will you take your description and add and issue for each feature request? It's actually much quicker that way.
